lib/ae_mdw_web/controllers/dex_controller.ex

```diff
@@ -48,16 +46,20 @@ defmodule AeMdwWeb.DexController do
   def swaps(%Conn{assigns: assigns} = conn, %{"from_symbol" => token_symbol}) do
     %{state: state, pagination: pagination, cursor: cursor} = assigns
 
-    with {:ok, create_txi} <- validate_token(token_symbol),
+    with {:ok, create_txi} <- Dex.validate_token(token_symbol),
```

Comment:
I meant removing validations from the controller as well. I wouldn't want the controller to know about the `create_txi` or any other dex internal concept. Controllers should all look like this
```elixir
def route(conn, params) do
  with {:ok, something} <- Context.fetch_something(state, ...) do
    Util.paginate(conn, something, ...)
  end  
end
```